src/fastcontext/context.c

```diff
@@ -60,7 +71,9 @@ void INTERNAL qt_makectxt(uctxt_t *ucp,
 #if 0
     memcpy(sp, &argc + 1, argc * sizeof(uintptr_t));
 #else
+QT_SUPPRESS_BOUNDS_CHECK_WARNINGS
     *(uintptr_t*)sp = *(uintptr_t*)((&argc)+1);
```

Comment:
Most of these changes are straightforward. This one's unusual though. I wasn't able to figure out how to get around the warning since it's technically correct to warn about an out-of-bounds access here. As far as I can tell though, there's just not a better way to get the address of the varargs than what this is already doing so instead I suppressed the warning.

Reply:
Actually, thinking about this some more, it's not actually forwarding the address of the varargs, just the value, so this *can* actually be done without the out of bounds access. I'll fix that.

Reply:
Okay, getting access to the varargs through the standard mechanisms appears to have worked fine for resolving that warning so it no longer needs to be suppressed and this is good to go.